packages/contracts-bedrock/src/L1/L1CrossDomainMessenger.sol

```diff
@@ -53,4 +58,9 @@ contract L1CrossDomainMessenger is CrossDomainMessenger, ISemver {
     function _isUnsafeTarget(address _target) internal view override returns (bool) {
         return _target == address(this) || _target == address(PORTAL);
     }
+
+    /// @inheritdoc CrossDomainMessenger
+    function paused() public view override returns (bool) {
```

Comment:
Odd that this didn't trigger an abi snapshot check failure in CI as snapshots haven't been updated.
When you run` pnpm abi-snapshot` in packages/contracts-bedrock, are any files under `packages/contracts-bedrock/snapshots` updated for you?

Reply:
hmmm, yes, I just ran it and added those files in 053066a.

Reply:
I'll will fix the CI check in another PR.